csdc-6/adcs-simulation/cpp/src/Messenger.cpp

```diff
@@ -188,7 +188,7 @@ void Messenger::append_csv_output(sim_config state, timestamp time)
             output_file << state.satellite.alpha_b.x() << "," << state.satellite.alpha_b.y() << "," << state.satellite.alpha_b.z() << ",";
 
             output_file << state.accelerometer.measurement.x() << "," << state.accelerometer.measurement.y() << "," << state.accelerometer.measurement.z() << ",";
-            output_file << state.gyroscope.measurement.x()     << "," << state.gyroscope.measurement.y()     << "," << state.gyroscope.measurement.z()     << ",";
+            // output_file << state.gyroscope.measurement.x()     << "," << state.gyroscope.measurement.y()     << "," << state.gyroscope.measurement.z()     << ",";
```

Comment:
do we still want this?

Reply:
Yes - again just for future reference until the output is finalized.